dbsnp-importer/src/test/resources/dbsnp-data.sql

```diff
@@ -127,6 +127,19 @@ INSERT INTO subind VALUES (12070,24939615,6483,NULL,NULL,93,NULL);
 INSERT INTO subind VALUES (12069,26011364,6482,NULL,NULL,58,NULL);
 INSERT INTO subind VALUES (12069,26011365,6482,NULL,NULL,4211,NULL);
```

Comment:
`EBISAMPLE1` and `EBISAMPLE2` were artificially added but now that we can use existing batches you can get rid of them.